printrun/stlview.py

```diff
@@ -90,7 +90,7 @@ def __init__(self, parent, size, id = wx.ID_ANY,
                                         circular = circular)
         self.dist = max(self.build_dimensions[0], self.build_dimensions[1])
         self.basequat = [0, 0, 0, 1]
-        wx.CallAfter(self.forceresize)
+        wx.CallAfter(self.forceresize) #why needed
```

Comment:
This was added to work around a bug where the view would not update initially. I am not sure if it's needed anymore, can you check if things still look okay when it's removed?

Reply:
Removing it does not change anything in Ubuntu/wxWidgets 3.1.4.
Do you mean #1028? It is fixed in this PR.
I can remove it in separate commit if you like to experiment.